packages/matter-node.js/test/persistence/PersistenceTest.ts

```diff
@@ -0,0 +1,96 @@
+/**
+ * @license
+ * Copyright 2022-2023 Project CHIP Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { StorageInMemory } from "../../src/persistence/StorageInMemory";
+import { Persistence } from "../../src/persistence/Persistence";
+import * as assert from "assert";
+import { SupportedStorageTypes } from "../../src/persistence/JsonConverter";
```

Comment:
What is the JsonConverter used for now?  I thought we aren't using JSON format for storage anymore.

Reply:
It is still used to convert the data into strings - check the node localstorage class.